### PR TITLE
Only show format if not null

### LINKED ISSUE #1980: dials.show shows the wrong format class
... which just caused some red herring hunting with @noemifrisina as it really confused me!

```
Grey-Area-2 nxs :) $ dials.import nexus_master.h5 
DIALS (2018) Acta Cryst. D74, 85-97. https://doi.org/10.1107/S2059798317017235
DIALS 3.dev.624-g142c56f76
The following parameters have been modified:

input {
  experiments = <image files>
}

--------------------------------------------------------------------------------
  format: <class 'dxtbx.format.FormatNexusEigerDLS.FormatNexusEigerDLS'>
  num images: 360
  sequences:
    still:    0
    sweep:    1
  num stills: 0
--------------------------------------------------------------------------------
Writing experiments to imported.expt
Grey-Area-2 nxs :) $ dials.show imported.expt 
DIALS (2018) Acta Cryst. D74, 85-97. https://doi.org/10.1107/S2059798317017235
The following parameters have been modified:

input {
  experiments = imported.expt
}

Experiment 0:
Format class: FormatMultiImage
Experiment identifier: 3cd382b0-bd16-43bb-a102-90a3eac4c24b
Image template: /private/tmp/nxs/nexus_master.h5
```

`dials.import` reports the true format class; `dials.show` reports a different one. Either it should report the same or nothing 😉

### PR DESCRIPTION
**Pull request by graeme-winter**
i.e. not Format or FormatMultiImage - used to just check was not Format.
Avoids confusion in looking at dials.show output.

Fixes #1980

### CODE REVIEW

File: command_line/show.py [hunk 1]
```diff
@@ -250,7 +250,7 @@ def show_experiments(experiments, show_scan_varying=False):
     for i_expt, expt in enumerate(experiments):
         text.append("Experiment %i:" % i_expt)
         format_class = expt.imageset.get_format_class()
-        if format_class.__name__ != "Format":
+        if format_class.__name__ not in ["Format", "FormatMultiImage"]:
```
Comment (ndevenish) [inline command_line/show.py:253]:
```suggestion
        if not format_class.abstract():
```
Would this be more robust?

Comment (Anthchirp) [reply command_line/show.py:253]: Possibly, but if not:
```suggestion
        if format_class.__name__ not in {"Format", "FormatMultiImage"}:
```

Comment (graeme-winter) [reply command_line/show.py]: `abstract()` not always defined 🤔

and it turns out `FormatMultiImage` is not abstract, either

Comment (graeme-winter) [reply command_line/show.py]: Done subsequent suggestion instead.

Comment (ndevenish) [reply command_line/show.py]: oh ffs it's `is_abstract`

Comment (graeme-winter) [reply command_line/show.py]: > oh ffs it's `is_abstract`

Probably yeh, will try that later

**Comment by ndevenish** [comment]
This test is now failing because of https://github.com/cctbx/dxtbx/pull/463

**Pull request merged**